src/coreclr/jit/optimizer.cpp

```diff
@@ -2566,6 +2572,23 @@ void Compiler::optFindNaturalLoops()
         fgUpdateChangedFlowGraph(computePreds);
     }
 
+    if (false /* pre-header stress */)
```

Comment:
I assume this will be enable-able in a follow-on change?

Reply:
Enabling this will depend on fixing loop cloning. I opened https://github.com/dotnet/runtime/issues/62665 to track.